android/app/src/main/kotlin/io/customer/amiapp_flutter/AmiAppLogger.kt

```diff
@@ -1,28 +1,58 @@
 package io.customer.amiapp_flutter
 
+import android.util.Log
+
+import io.customer.sdk.util.CioLogLevel
 import io.customer.sdk.util.Logger
 
-internal class AmiAppLogger(
-    private val logger: Logger,
-) : Logger {
+class AmiAppLogger : Logger {
```

Comment:
I feel like this class can be simplified, Instead of using preferredLogLevel to store the user-defined log level and fallback to CioLogLevel.DEBUG, you can set a default log level value of CioLogLevel.DEBUG directly in the constructor parameter.